lib/dap/proto/ldap.rb

```diff
@@ -25,8 +49,15 @@ def self.decode_elem_length(data)
       len_bytes = length - 128
       return unless data.length > len_bytes + 2
 
-      if len_bytes == 2
+      if len_bytes == 1
+        length = data.byteslice(2, len_bytes).unpack('C')[0]
+      elsif len_bytes == 2
         length = data.byteslice(2, len_bytes).unpack('S>')[0]
```

Comment:
For here and in the fall-through `else` on line 61, is forcing the endianness correct here?  Are these explicitly stored as big endian?